modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -1032,6 +1032,26 @@ def test_insert(data):
     )
 
 
+def test_insert_4407():
```

Comment:
Shouldn't we also add a test to ensure that the bug you found fails in both Modin and pandas?
i.e. that inserting multiple columns via an np.array fails?

Reply:
@RehanSD thanks for review! I forgot after local testing to return an array case with multiple columns. Updated test.